build.yaml

```diff
@@ -168,10 +176,21 @@ common_cli:
     # https://gitlab.com/gitlab-org/gitlab-ci-multi-runner/issues/20
     # So we will place the virtualenv in /tmp/v instead
     #
-    sudo rm -rf \$venv
-    sudo rm -f results.xml
-    sudo rm -f trial.log
-    sudo rm -rf _trial_temp/
+    if [ 'OSX' == "\$(os)" ]; then
+      # OSX doesn't allow us to sudo (no tty)
+      rm -f results.xml
+      rm -f trial.log
+      rm -rf _trial_temp/
+      rm -rf \$venv
```

Comment:
Same as another comment, let's make it so we're on a clean box and these lines aren't necessary